scripts/mkindex.sh

```diff
@@ -66,7 +66,7 @@ $MOZSEARCH_PATH/scripts/objdir-mkdirs.sh
 
 date
 
-$MOZSEARCH_PATH/scripts/process-chrome-map.py $GIT_ROOT $INDEX_ROOT/url-map.json $INDEX_ROOT/*.chrome-map.json || handle_tree_error "process-chrome-map.py"
+$MOZSEARCH_PATH/scripts/process-chrome-map.py $GIT_ROOT $INDEX_ROOT/aliases/url-map $INDEX_ROOT/*.chrome-map.json || handle_tree_error "process-chrome-map.py"
```

Comment:
Is there a reason to not have the file have a `.json` extension anymore?  It seems like the file is still legal JSON?